static/app/views/replays/detail/network/index.tsx

```diff
@@ -208,6 +245,28 @@ function NetworkList() {
                   />
                 )}
               </AutoSizer>
+              {sortConfig.by === 'startTimestamp' && showJumpUpButton ? (
+                <Button
+                  onClick={handleClick}
+                  aria-label={t('Jump Up')}
+                  priority="primary"
+                  size="xs"
+                  style={{position: 'absolute', justifySelf: 'center', top: '28px'}}
```

Comment:
Can you put these common position & justify-self properties into a custom button component? Something like this:

```
<UpDownButton
  onClick={handleClick}
  aria-label={t('Jump Up')}
  style={{top: '28px'}}
  ...

<UpDownButton
  onClick={handleClick}
  aria-label={t('Jump Down')}
  style={{top: '5px'}}
  ...

...

const UpDownButton = styled('div')`
  position: absolute;
  justify-self: center;
`;
```